src/commands/revenue.js

```diff
@@ -100,7 +116,7 @@ class Report {
     // find the earliest date a submission was assigned. This date is needed to
     // get the correct number of days of the period, so we can calculate the
     // average daily earnings for the period.
-    if (this.startDate === '2014-01-01') {
+    if (moment(this.startDate).format('DD') === '01') {
```

Comment:
I've increased the situations where it will calculate the first day on the job. But I feel that there is an easier way to do this, but I don't know how.